src/cljs/main/broadfcui/header.cljs

```diff
@@ -42,7 +42,7 @@
                             :color "#000" :textDecoration "none"
                             :padding "0.5rem" :border style/standard-line}
                     :data-test-id "account-dropdown"}
-              (-> auth2 (.-currentUser) (.get) (.getBasicProfile) (.getEmail))
+              [:div {:style {:display "inline-block"} :data-test-id "account-dropdown-email"} (-> auth2 (.-currentUser) (.get) (.getBasicProfile) (.getEmail))]
```

Comment:
Any reason not to use a `span` instead of a `div`? The `span` should be inherently inline so you wouldn't need the `inline-block` style.